RetrospectiveExtension.Frontend/config/environment.tsx

```diff
@@ -1,5 +1,5 @@
 const environment = {
-  CollaborationStateServiceUrl: 'https://retrospectivesbackendservice.azurewebsites.net',
+  CollaborationStateServiceUrl: '<collaboration state service url>',
```

Comment:
It would be cleaner if this value is moved appsettings.json. Thoughts?

Reply:
I assume you meant `package.json`, because `appsettings.json` file is in the `Backend` project.

if that's the case, I'd be hesitant to put an implementation detail into the project definition file.

I was thinking to move this variable into an environment variable, but since it's used in a `tsx` file, it's not easy to replace the value with an environment variable 😞